packages/cardano-services/src/Http/schema.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Thinking about this point a bit more: this can be helpful only if we plan to support multiple API versions with one `provider-server`; if, on the other hand, our target is to support only a single API version with one `provider-server` and handling versions at routing level, this proposal looses its value.

Reply:
Thanks for the prompt, and yes you're right, for now we'll just be supporting one version.